akka-actor/src/main/scala/akka/actor/ActorPath.scala

```diff
@@ -98,7 +106,7 @@ object ActorPath {
  * when comparing actor paths.
  */
 @SerialVersionUID(1L)
-sealed trait ActorPath extends Comparable[ActorPath] with Serializable {
+sealed abstract class ActorPath extends Comparable[ActorPath] with Serializable {
```

Comment:
Yes, I know; I’m just keeping a watchful eye on these as we will have to correct all the mistakes in a few weeks time anyway.